### PR TITLE
Sensu webhooks

### PR DESCRIPTION
**Pull request by novotnyjakub**
This is a spec'd version of Sensu webhook implementation. But it may need to change slightly when it's actually being integrated with Sensu. But that should only have implication on the `payload` getting through.

### CODE REVIEW

File: services/sensu_webhook.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class SensuWebhook
+  REDIS_KEY_PREFIX = 'sensu'
+
+  def initialize(payload)
+    @payload = payload
+  end
+
+  def process
+    service_id = @payload['key']
+    redis_key = "#{REDIS_KEY_PREFIX}/#{service_id}"
```
Comment (james) [inline services/sensu_webhook.rb]: My knowledge of redis isn't great, but for other services, aren't we using a colon as a divider not a forward slash?

https://github.com/ministryofjustice/2nd-line-support/blob/master/models/traffic_spike.rb#L8

Comment (novotnyjakub) [inline services/sensu_webhook.rb]: Can be, I just followed what I did for Pingdom and there I used `/`. But there isn't a particular reason, so I'll change both and update the PR.

**Comment by novotnyjakub** [comment]
PR updated to follow the Redis key convention

**Comment by james** [comment]
Cool, thanks. Happy for me to merge now?

**Comment by novotnyjakub** [comment]
:+1:

**Comment by james** [comment]
BAM. thanks.

**Pull request merged**